src/swaps/liquality/LiqualitySwapProvider.ts

```diff
@@ -506,39 +510,6 @@ export class LiqualitySwapProvider extends SwapProvider {
     };
   }
 
-  private async fundSwap({ swap, network, walletId }: NextSwapActionRequest<LiqualitySwapHistoryItem>) {
```

Comment:
As mentioned offline, we need the approval step separately in order to track it.